src/main/io/rcdevice.h

```diff
@@ -150,6 +152,12 @@ typedef struct {
     rcdeviceRespParseFunc parseFunc;
 } rcdeviceWaitingResponseQueue;
 
+typedef struct {
+    uint8_t command;
+    uint8_t data[255];
+    uint8_t dataLength;
+} runcamDeviceRequest_t;
```

Comment:
Do you have a specification for the format / commands to be used by these requests?